.github/ISSUE_TEMPLATE/issue-default.yml

```diff
@@ -1,32 +1,32 @@
-name: Standard Issue Tempalte
-description: Create an Issue
-title: "[Issue]:"
-labels: ["research"]
-assignees:
-  - kitsushadow
+name: Issue
+description: Provide a title for the issue you are describing 
+title: "Please provide a clear title"
+labels: ["review"]
+# assignees:
+#   - kitsushadow
 body:
   - type: markdown
     attributes:
       value: |
-        Please fill this out best you can
+        Describe an issue, idea, task, bug, or story.
   - type: textarea
     id: why
     attributes:
       label: Why or Acceptance Criteria
-      description: What is the reason we determined we need an issue.
+      description: What is the reason it was determined this is an issue.
       placeholder: This issue is a product of...
-      value: "While working on X or researching Y it was discovered that Z was an issue that need captured."
+      value: "While working on X or researching Y it was discovered that Z was an issue that needed captured."
```

Comment:
With bug reports and user stories separated into different templates, I recommend doing something like this for the user story template, to prefill the format we might want folks to answer in

```suggestion
      value: |
        As a <!--who-->
        I want <!--what-->
        So that <!--why-->
```

Reply:
Not sure if it will use the `<!--comment-->` md syntax, so we can leave that out if not

Reply:
Then we could add an additional textarea of acceptance criteria, perhaps?

I don't have strong preferences on format, but I've seen folks enjoy the given / when / then format in the past, so I'll add that as a suggestion, e.g:

> ## Example Acceptance Criteria – Given/When/Then
> 
> ### User story:
> 
> As a product manager.
> 
> I want to score potential ideas.
> 
> So that I can decide what to include on my product roadmap.
>
> ### Acceptance criteria for that user story could be:
> 
> Scenario: The product manager adds potential ideas and ranks the best ideas based on benefit versus cost.
> 
> Given that I have added two or more ideas and scored them using the Benefit vs Cost scoring model
> 
> When I click the Rank button
> 
> Then ideas are sorted with the top-scoring ideas at the top.
> 

- [source](https://www.productplan.com/glossary/acceptance-criteria/)